app/components/SQLTable.react.js

```diff
@@ -20,10 +22,10 @@ export default function SQLTable(props) {
                     flexgrow={1}
                     width={200}
                     header={<Cell>{colName}</Cell>}
-                    cell={({rowIndex, ...props}) => (
+                    cell={({rowIndex, ...cellProps}) => (
                         <Cell
                             height={20}
-                            {...props}
+                            {...cellProps}
```

Comment:
where is `cellProps` coming from?

Reply:
If I understand it correctly, `...cellProps` is *collecting* the arguments `width` and `height` that are passed into the callback function that is creating the `Cell`. See [FixedDataTable](http://facebook.github.io/fixed-data-table/api-cell.html) and the [Rest parameters syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters).

Reason for renaming from `props` to `cellProps` is that `props` is already defined in outer scope. See ESLint [no-shadow](https://eslint.org/docs/rules/no-shadow) rule.